plugins/flytekit-dask/flytekitplugins/dask/models.py

```diff
@@ -0,0 +1,129 @@
+from typing import Optional
+
+from flyteidl.plugins import dask_pb2 as _dask_task
+
+from flytekit.models import common as _common
+from flytekit.models import task as _task
+
+
+class Scheduler(_common.FlyteIdlEntity):
+    """
+    Configuration for the scheduler pod
+
+    :param image: Optional image to use.
+    :param resources: Optional resources to use.
+    """
+
+    def __init__(self, image: Optional[str], resources: Optional[_task.Resources]):
```

Comment:
```suggestion
    def __init__(self, image: Optional[str]=None, resources: Optional[_task.Resources]=None):
```

Reply:
Fixed in dab1bc635f91041e849e0bdfc03e187846a41de7